cmd/operator-sdk/add/api.go

```diff
@@ -127,7 +127,7 @@ func apiRun(cmd *cobra.Command, args []string) error {
 		return projutil.ErrUnknownOperatorType{}
 	}
 	// Verify the incoming flags.
-	if err := apiFlags.VerifyCommonFlags(operatorType); err != nil {
+	if err := apiFlags.VerifyAnsibleFlags(operatorType); err != nil {
```

Comment:
I need to revert that. We will be able to just change/remove after impl the api plugin as well.